platforms/jvm/testing-jvm-infrastructure/src/main/java/org/gradle/api/internal/tasks/testing/failure/TestFailureMapper.java

```diff
@@ -80,9 +81,29 @@ public boolean supports(Class<?> cls) {
     /**
      * Utility method to invoke a method on an object by reflective means.
      */
-    protected static <T> T invokeMethod(Object obj, String methodName, Class<T> targetClass) throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
+    @Nullable
+    protected static <T> T invokeMethod(@Nullable Object obj, String methodName, Class<T> targetClass) throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
```

Comment:
Does it make sense to check if `obj` is null before calling this method instead of making this method accept `null`?